WordPressKit/HTTPRequestBuilder.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
 
+/// A builder type that adding HTTP request data onto an URL.
```

Comment:
nit: typo?

Reply:
🙈 Fixed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/694/commits/59bd717519e4ce71a4091193bd2015ea59454607.